CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
+## 1.8.0 _(Not Yet Released)_
+
+### Code Actions
```

Comment:
Also, I wouldn't put this under "Code Actions". Those are for code fixes and refactorings that appear in the light bulb menu. I'd actually give this it's own subheading: "Go to Implementation".